crates/compiler/builtins/roc/Dict.roc

```diff
@@ -208,3 +210,413 @@ insertIfVacant = \dict, key, value ->
         dict
     else
         Dict.insert dict key value
+
+# We have decided not to expose the standard roc hashing algorithm.
+# This is to avoid external dependence and the need for versioning.
+# For now, it will just live within the Dict file.
+# If we add non-exposed roc standard library files, this should be moved to it's own file.
```

Comment:
I think we could add non-exposed roc std library files now if you'd like, but given the small size of the Hasher impl here, I think it's good to leave as is if you do.

Reply:
Removed the comment. I think it makes sense to just leave it all here. It is not like the function names are likely to conflict.